home-assistant/addons/sbam/config.json

```diff
@@ -41,6 +44,9 @@
     "batt_reserve_end_hr": "match(^$|^([01]?[0-9]|2[0-3]):[0-5][0-9]$)",
     "defaults": "bool",
     "reset": "bool",
-    "debug": "bool"
+    "debug": "bool",
+    "cache_forecast": "bool",
```

Comment:
add and export these three new params here: https://github.com/atbore-phx/sbam/blob/main/home-assistant/addons/sbam/run.sh

this file creates the env vars and exposes them to the docker container when you use the Home Assistant addon

Reply:
Done

Reply:
I noticed we’re using both `cache_file` and `cache_file_name` for the same variable. To keep naming consistent and avoid confusion, could we stick with cache_file_name?

Reply:
Thanks. That was an error. It should always have been cache_file_name